lib/repositories/stats/stats_repository.dart

```diff
@@ -82,10 +89,174 @@ class StatsRepositoryImpl extends StatsRepository {
       await pref.remove('secsListened');
       await pref.remove('numSessions');
       await pref.remove('longestStreak');
-    } catch (e) {
+    } catch (err) {
+      await Sentry.captureException(
+        err,
+        stackTrace: err.toString(),
+      );
       rethrow;
     }
   }
+
+  @override
+  Future<void> addDummyStats() async {
+    var sharedPreferences = await SharedPreferences.getInstance();
+    var ids = [
+      390,
+      9,
+      20,
+      9,
+      2,
+      87,
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9,
+      10,
+      11,
+      12,
+      13,
+      14,
+      15,
+      16,
+      17,
+      18,
+      19,
+      20,
+      21,
+      22,
+      23,
+      24,
+      25,
+      26,
+      27,
+      28,
+      29,
+      30,
+      31,
+      32,
+      33,
+      34,
```

Comment:
What is all this? Do we need this in develop?

Reply:
I added this code for testing long stats transfer. Let me remove it now